test/rsrc/convert_stub.py

```diff
@@ -20,8 +27,11 @@ def convert(in_file, out_file, tag):
     # On Windows, use Unicode paths. (The test harness gives them to us
     # as UTF-8 bytes.)
     if platform.system() == 'Windows':
-        in_file = in_file.decode('utf8')
-        out_file = out_file.decode('utf8')
+        if not six.PY2:
+            in_file = in_file.encode(arg_encoding())
+            out_file = out_file.encode(arg_encoding())
+        in_file = in_file.decode('utf-8')
+        out_file = out_file.decode('utf-8')
```

Comment:
if you mean no encode/decode at all, then no. That was the very first thing i tried. My first iteration removed the platform check, and it couldn't find the files.  It passed the escape strings right along, and `open` didn't like that.